### PR TITLE
feat(grid): add breakpoint-specific alignment and order classes

### PR DESCRIPTION
**Pull request by pauleustice**
# Description

Ports over various grid classes from the original http://flexboxgrid.com/ styles. Allows ordering and alignment at different viewport sizes.

## Requirements

Storybook link: (once netlify has deployed link provide a link to the component)

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [x] I have provided a story in storybook to document the changes

### CODE REVIEW

File: projects/canopy/src/styles/grid.scss [hunk 1]
```diff
@@ -49,6 +49,53 @@ $columns: 12;
   max-width: 100%;
 }
 
+@mixin lg-grid-alignment($breakpoint) {
+  // Alignment and distribution
+
+  .start-#{$breakpoint} {
```
Comment (elenagarrone) [inline projects/canopy/src/styles/grid.scss:55]: Looks good but could you add the `lg-` prefix to the classes please?

File: projects/canopy/src/styles/grid.scss [hunk 2]
```diff
@@ -49,6 +49,53 @@ $columns: 12;
   max-width: 100%;
 }
 
+@mixin lg-grid-alignment($breakpoint) {
```
Comment (elenagarrone) [inline projects/canopy/src/styles/grid.scss:52]: I would probably create directives for these too

File: projects/canopy/src/styles/grid.scss (hunk 1 above)
Comment (pauleustice) [reply projects/canopy/src/styles/grid.scss]: Done

File: projects/canopy/src/styles/grid.scss (hunk 2 above)
Comment (pauleustice) [reply projects/canopy/src/styles/grid.scss:52]: See https://github.com/Legal-and-General/canopy/issues/441

**Review by elenagarrone — APPROVED** [review]
Thank you!

**Review by andy-polhill — APPROVED** [review]
Thanks @pauleustice. I had a slight hesitancy around the extra size it might add, but I can see there is a need for it. It could be in the future we try and break up css, or even just have a build size allowance that runs in the pipeline so we have visibility over the size

**Pull request merged**